src/app/components/planner/Planner.tsx

```diff
@@ -209,10 +205,6 @@ function Modals() {
   return (
     <>
       <CourseInfoModal />
```

Comment:
Replace the `<Modals />` in here with `<CourseInfoModal />` since it's the only component